.github/workflows/test.yaml

```
@@ -95,3 +95,67 @@ jobs:
       - uses: ./.github/actions/setup-env
       - name: Run optimized tests
         run: tox -e optimized
+
+  mutation-test:
+    runs-on: ubuntu-latest
+    needs: static
+    if: github.event_name == 'pull_request'
+    strategy:
+      matrix:
+        fork: [frontier, homestead, tangerine_whistle, spurious_dragon, byzantium, constantinople, istanbul, berlin, london, paris, shanghai, cancun, prague, osaka]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+      - name: Install project with test dependencies
+        run: pip install -e '.[test]'
+      - name: Install mutmut
+        run: pip install mutmut==3.3.0
+      - name: Apply mutmut patch
+        run: |
+          git clone https://github.com/souradeep-das/mutate-execution-specs.git
+          cp mutate-execution-specs/mutmut.patch .
+          MUTMUT_FILE_TO_PATCH=$(python3 -c "import mutmut, os; print(os.path.join(os.path.dirname(mutmut.__file__), '__main__.py'))")
+          patch $MUTMUT_FILE_TO_PATCH < mutmut.patch
+      - name: Find changed files for fork
+        id: changed
+        run: |
+          git fetch origin ${{ github.event.pull_request.base.ref }}
+          CHANGED_FILES=$(git diff --name-only origin/${{ github.event.pull_request.base.ref }} ${{ github.sha }} | grep "^src/ethereum/${{ matrix.fork }}/" || true)
+          echo "changed_files<<EOF" >> $GITHUB_OUTPUT
+          echo "$CHANGED_FILES" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
```

```suggestion
          CHANGED_FILES=$()
          echo "changed_files<<EOF" >> $GITHUB_OUTPUT
          ( git diff --name-only origin/${{ github.event.pull_request.base.ref }} ${{ github.sha }} | grep "^src/ethereum/${{ matrix.fork }}/" || true ) >> $GITHUB_OUTPUT
          echo "EOF" >> $GITHUB_OUTPUT
```

Can avoid storing the list in a temporary variable.